src/app/filters/filter-query.ts

```diff
@@ -7,4 +7,6 @@
 export class FilterQuery {
   id?: string;
   value: string;
+  imageUrl?: string;
+  iconClass?: string;
```

Comment:
Looking at the UXD mockups, I believe also need to support icons and images for the main filter options as well. That is, in addition to filter "queries".